<HTML><HEAD><TITLE> RESULTS</TITLE> </HEAD><BODY BGCOLOR="FFFFF0"><b></b> 
<h2>
<a href=http://dx.doi.org/10.1016/j.jbi.2007.02.003>
A new framework for identifying combinatorial regulation of 
transcription factors: a case study of the yeast cell-cycle </A></b> </h2> 
<br> 

<b><A HREF="mailto:jw2256@columbia.edu">Junbai Wang</a>* </b> <br><br> 
<b>Biological Science Department* Columbia Univesrity New York, 10025 USA 
</b> <p> <br> By integrating heterogeneous functional genomic datasets, we 
have developed a new framework for detecting combinatorial control of 
gene expression, which includes estimating transcription factor activities 
using a singular value decomposition method and reducing high-dimensional 
input gene space by considering genomic properties of gene clusters. The 
prediction of cooperative gene regulation is accomplished by either 
Gaussian Graphical Models or Pairwise Mixed Graphical Models. The proposed 
framework was tested on yeast cell cycle datasets: (1) 54 known yeast cell 
cycle genes with 9 cell cycle regulators and (2) 676 putative yeast cell 
cycle genes with 9 cell cycle regulators. The new framework gave promising 
results on inferring TF-TF and TF-gene interactions. It also revealed 
several interesting mechanisms such as negatively correlated 
protein-protein interactions and low affinity protein-DNA interactions 
that may be important during the yeast cell cycle. The new framework may 
easily be extended to study other higher eukaryotes.

</br>
